### PR TITLE
Addresses Request for Documentation: Typo in plugin webapp developer documentation #489

### PR DESCRIPTION
**Pull request by RohitJain13**
#### Summary
Fixing import paths in plugin webapp documentation.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-developer-documentation/issues/489

### CODE REVIEW

File: site/content/extend/plugins/webapp/reference.md [hunk 1]
```diff
@@ -161,8 +161,8 @@ Converts HTML to React components.
 A short usage example of a `PostType` component using the post utility functions to format text.
 
 ```javascript
-const React = window.react;
-const PostUtils = window['post-utils']; // import the post utilities
+const React = window.React;
+const PostUtils = window.PostUtils; // import the post utilities
 import PropTypes from 'prop-types';
```
Comment (larkox) [inline site/content/extend/plugins/webapp/reference.md]: Could you update this line to use the same behaviour as the top other ones?
```suggestion
const PropTypes = window.PropTypes;
```

**Comment by mickmister** [comment]
I'm not sure I like the approach of explicitly accessing the modules from `window`. I believe that makes the typings for the modules not work with an IDE. I think we should instead advise to make sure the externals are appropriately specified in `webpack.config.js`:

https://github.com/mattermost/mattermost-plugin-starter-template/blob/5d32943c364e38b1c6c5503003b6b392f529c65d/webapp/webpack.config.js#L30-L36

The one exception being `PostUtils`, because that is not an npm module. @larkox What are your thoughts on this?

**Comment by larkox** [comment]
@mickmister  Never thought about the IDE. Probably we should bring someone from Toolkit to the discussion, to clarify why this page of documentation was saying this, and what are the real benefits. If this provides no clear benefit, why not remove these things from webapp and remove this part of the documentation?

**Comment by mickmister** [comment]
@ali-farooq0 Do you have any thoughts on my approach for the documentation?

**Comment by ali-farooq0** [comment]
@mickmister true, like you rightly pointed out `PostUtils` can't get added to the webpack, at least not without some trickery. Also did you mean to make the change in webpack as a means for documentation?

Because I think regardless of how those modules are exposed, the docs should still be updated, no? Otherwise we'd have a broken example as @larkox pointed out in his post [here](https://community.mattermost.com/core/pl/iy47fesyht8kbk7qwkt7br46xh)

**Comment by mickmister** [comment]
@ali-farooq0 Yes the docs should be updated, just discussing what it should change to. I think it's safe to assume that a developer using these docs is working with a plugin that is using the `import` syntax. I don't think I've seen a plugin written to explicitly import React through `window`.

The docs already give a nod to the webpack externals documentation, so I think it's reasonable to assume that it's fair game to use in the code example. Please see my suggested change.

> Also did you mean to make the change in webpack as a means for documentation?

Nope :) I think the webpack config is fine, but we should encourage using the standard import syntax for known node modules.

**Comment by mickmister** [comment]
@RohitJain13 Thank you for your contribution and your speedy response!

**Comment by ali-farooq0** [comment]
> Nope :) I think the webpack config is fine, but we should encourage using the standard import syntax for known node modules.

Gotcha, fair point. I agree, for the module imports we should be encouraging `import` syntax 👍 Not sure why didn't see your suggestion before but yes your suggestions looks good 👍

**Comment by mickmister** [comment]
> Not sure why I didn't see your suggestion before

@ali-farooq0 I hadn't posted it before. I made the suggestion when I posted my  reply :smile:

**Pull request merged**